helpers/filterSeats.js

```diff
@@ -58,12 +59,35 @@ module.exports = function(dust) {
       });
     }
 
+    // use Transformer show config to add Gold, Silver or Bronze to packageRate
+    function assignColoursToBands(packageRate) {
+      // assign gold, silver, bronze to packages depending on their current priceBand
+      _.forEach(bandColours, function(sectionValue, sectionKey) {
+        // check for match between transformer and existing data
+        // check for section that might not have full title ( e.g. Grand Circle in Grand Circle (Left) )
+        if (ticketRates[packageRate.ids].section === sectionKey || ticketRates[packageRate.ids].section.indexOf( sectionKey ) > -1 ) {
+          // match transformer config to existing seat section
+          _.forEach(sectionValue, function(bandValue, bandKey) {
+            if (ticketRates[packageRate.ids].priceBand === bandKey) {
+              // match transformer config to existing priceBand
+              ticketRates[packageRate.ids].colour = bandValue;
```

Comment:
can we bin one of these assigns please, don't want the same info twice on the api reply